data/keystore/src/main/java/tech/pegasys/artemis/bls/keystore/builder/Pbkdf2ParamBuilder.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.artemis.bls.keystore.builder;
+
+import org.apache.tuweni.bytes.Bytes32;
+import tech.pegasys.artemis.bls.keystore.model.Pbkdf2Param;
+import tech.pegasys.artemis.bls.keystore.model.Pbkdf2PseudoRandomFunction;
+
+public final class Pbkdf2ParamBuilder {
```

Comment:
Looks like this only ever used in the tests and so should be moved out of prod code

Reply:
all the builders are now gone.